web/src/lib/components/assets/thumbnail/image-thumbnail.svelte

```diff
@@ -1,27 +1,38 @@
 <script lang="ts">
-	import { imageLoad } from '$lib/utils/image-load';
+	import { onMount } from 'svelte';
+	import { lazyLoad } from 'unlazy';
+	import * as Buffer from 'buffer';
 
 	export let url: string;
 	export let altText: string;
 	export let heightStyle: string | undefined = undefined;
 	export let widthStyle: string;
+	export let thumbhash: Buffer;
 	export let curve = false;
 	export let shadow = false;
 	export let circle = false;
-	let loading = true;
-</script>
-
-<img
+  
+	let imageElement: HTMLImageElement;
+  
+	onMount(() => {
```

Comment:
Putting everything in `onMount` means SSR responses only show the thumbhash after loading JS. That will happens regardless right now, because of the intersection observer in `Thumbnail` but might be worth to explore if this can be made compatible with SSR.

Reply:
I tried doing it without the `onMount` but it caused some images to not load properly as the script would try to call `lazyLoad()` before the images existed.